CHANGES

```diff
@@ -13,10 +13,15 @@ Deprecated
 Features added
 --------------
 
+* #10523: HTML Theme: Expose the Docutils's version info tuple as a template
+  variable, ``docutils_version_info``. Patch by Adam Turner.
+
 Bugs fixed
 ----------
 
 * #10509: autosummary: autosummary fails with a shared library
+* #10523: HTML Theme: Fix double brackets on citation references in Docutils 0.18+.
+  Patch by Adam Turner.
```

Comment:
Is this a new format of CHANGES?

Reply:
I prefer capitalising HTML at the least, as it is an initialism -- I could leave "theme" lowercased?

A

Reply:
Okay, understandable.